packages/sdk-bridge/changelog.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0-rc.4
```

Comment:
Top level title and an empty unreleased section should be present. See the configuration changelog for example